app/components/post_draft/post_input/post_input.tsx

```diff
@@ -314,6 +323,7 @@ export default function PostInput({
             testID={testID}
             underlineColorAndroid='transparent'
             textContentType='none'
+            value={value}
```

Comment:
what side effect do we get by making this a controlled input?

Reply:
With the testing I have done, I haven't seen any. Not sure why it was not a controlled input before. But if you feel there is any potential issue, let me know and I will test it.

Reply:
for example typing quickly, does it cause any issues? what about composing large messages

Reply:
Nothing noticeable from my testing on the simulator (where I can write way faster than on mobile :sweat_smile:), neither on android or ios.